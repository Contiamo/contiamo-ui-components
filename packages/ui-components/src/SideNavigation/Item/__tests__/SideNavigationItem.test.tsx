import * as React from "react"
import { render } from "enzyme"

import ThemelessSideNavigationItem from "../SideNavigationItem"
import wrapDefaultTheme from "../../../utils/wrap-default-theme"

const SideNavigationItem = wrapDefaultTheme(ThemelessSideNavigationItem)

test("SideNavigationItem component renders", () => {
  const output = render(<SideNavigationItem>Hi, I'm an Item</SideNavigationItem>)
  expect(output).toMatchSnapshot()
})
