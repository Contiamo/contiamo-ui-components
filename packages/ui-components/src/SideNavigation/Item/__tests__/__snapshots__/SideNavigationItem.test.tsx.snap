// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideNavigationItem component renders 1`] = `
<div
  class="css-1vqcugq undefined"
  role="button"
  tabindex="-1"
>
  Hi, I'm an Item
</div>
`;
